data/graph_schemas/graph_config_predict_schema.yml

```diff
@@ -0,0 +1,238 @@
+nodes:
```

Comment:
Is this the same as `default_config_predict_schema.yml`?

Reply:
Yup, removed the file for that reason.